bip-0360.mediawiki

```diff
@@ -40,11 +40,11 @@ relies on post-quantum cryptographic (PQC) signature algorithms. By adopting PQC
 resistance without requiring a hard fork or block size increase.
 
 The vulnerability of existing Bitcoin addresses is investigated in
-[https://web.archive.org/web/20240715101040/https://www2.deloitte.com/nl/nl/pages/innovatie/artikelen/quantum-computers-
-and-the-bitcoin-blockchain.html this Deloitte report]. The report estimates that in 2020 approximately 25% of the
-Bitcoin supply is held within addresses vulnerable to quantum attack. As of the time of writing, that number is now
-closer to 20%. Independently, Bitcoin developer Pieter Wuille [https://x.com/pwuille/status/1108085284862713856 reasons]
-even more addresses might be vulnerable, representing 5M to 10M bitcoin.
+[https://web.archive.org/web/20240715101040/https://www2.deloitte.com/nl/nl/pages/innovatie/artikelen/quantum-computers-and-the-bitcoin-blockchain.html this Deloitte report].
+The report estimates that in 2020 approximately 25% of the Bitcoin supply is held within addresses vulnerable to
+quantum attack. As of the time of writing, that number is now closer to 20%. Independently, Bitcoin developer Pieter
+Wuille [https://x.com/pwuille/status/1108085284862713856 reasons] even more addresses might be vulnerable, representing
+5M to 10M bitcoin.
```

Comment:
As [described](https://github.com/bitcoin/bips/pull/1670#discussion_r1896085309) above, please don’t reformat the entire paragraph when you change a single line. Moving all the line breaks makes it needlessly difficult to see what actually changed about the text.

Reply:
Sorry, I just noticed that it was over 120 characters in length, which was something you requested earlier.

Reply:
Ah, I see. Let me clarify: I would suggest that you generally aim for a limited line length so that suggestions left in review refer to a limited amount of text and it is easy to see what changed. When you add a word or two in a change that pushes the line to a slightly larger length, you can either just leave it a bit longer, or you break just that line into two lines, leaving the rest of the lines without changes. That way it is still easy to say what was changed about the text. If that leaves an occasional line a little longer or some lines shorter, it is still easy to review and doesn’t change the visual appearance of the final document.

If you instead reformat the entire paragraph, a diff will highlight everything as changed which makes it more time consuming to review.